site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -102,17 +116,42 @@ export const WorkspacesPageView: FC<
         <WorkspacesFilter error={error} {...filterProps} />
       </Stack>
 
-      <PaginationStatus
-        isLoading={!workspaces && !error}
-        showing={workspaces?.length ?? 0}
-        total={count ?? 0}
-        label="workspaces"
-      />
+      <TableToolbar>
+        {checkedWorkspaces.length > 0 && isWorkspaceBatchActionsEnabled ? (
```

Comment:
Suggestion: Maybe this can be simplified? `checkedWorkspaces.length` can only be > 0 if batch actions is enabled, so we could only check that?